common/src/format.rs

```diff
@@ -416,6 +416,18 @@ impl FormatSpec {
         }
     }
 
+    pub fn format_bool(&self, input: bool) -> Result<String, FormatSpecError> {
+        let x = match input {
+            true => 1,
+            false => 0,
+        };
```

Comment:
You could probably do something simpler like this:

```suggestion
        let x = u8::from(input);
```

(Just make sure Clippy is okay with it.)

Reply:
Looks good, clippy didnt complain locally at least.